nodes/GeneratorExp.py

```diff
@@ -0,0 +1 @@
+print(g for ip in [num for num in range(1, 9)])
```

Comment:
You don't need the call to print - just the parentheses should be enough.

Also, you don't need the nested list comprehension: `(num for num in range(10))` should work just fine, yeah?